common/dtos.go

```diff
@@ -52,3 +54,18 @@ type InitialAccountAPI struct {
 	StakingValue string            `json:"stakingvalue"`
 	Delegation   DelegationDataAPI `json:"delegation"`
 }
+
+type AlteredAccountTokenAPIResponse struct {
```

Comment:
I hate to say this, but - add comments (here and below)?

Reply:
added